bundles/core/src/test/java/com/adobe/cq/wcm/core/components/ModelsCommonsTest.java

```diff
@@ -24,7 +24,8 @@
     @Test
     public void testDefaultBehaviour() throws Exception {
         testDefaultBehaviour(new String[] {
-                "com.adobe.cq.wcm.core.components.models"
+                "com.adobe.cq.wcm.core.components.models",
+                "com.adobe.cq.wcm.core.components.models.oembed"
```

Comment:
Can be removed when oembed services are moved to `com.adobe.cq.wcm.core.components.services.oembed` and UnsupportedOperationException is not thrown